<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infotronic</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_estado_general">
       <attribute name="title">
        <string>Estado general</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>141</width>
          <height>191</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 158, 0);</string>
        </property>
        <property name="title">
         <string>Selector</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonManual">
           <property name="text">
            <string>Manual</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonTemporizado">
           <property name="text">
            <string>Temporizado</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAutomatico">
           <property name="text">
            <string>Automatico</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOk">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>731</width>
          <height>211</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Marcador</string>
        </property>
        <widget class="QLCDNumber" name="lcdHumedad">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>50</y>
           <width>191</width>
           <height>81</height>
          </rect>
         </property>
         <property name="intValue" stdset="0">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Humedad :</string>
         </property>
        </widget>
        <widget class="QLabel" name="Hora">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>311</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hora</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>60</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="fecha_horaria">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>61</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Hora :</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_tecla_2">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>160</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Enviar Hora</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdTemperatura">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>191</width>
           <height>81</height>
          </rect>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="intValue" stdset="0">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>50</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>cÂ°</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Temperatura</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>240</y>
          <width>701</width>
          <height>241</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>230</y>
          <width>701</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 158, 0);</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Log de Datos Enviados</string>
         </attribute>
         <widget class="QListWidget" name="listEnvio">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>751</width>
            <height>341</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: black; color: green;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Log de Datos Recibidos</string>
         </attribute>
         <widget class="QListWidget" name="listErrores">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>751</width>
            <height>331</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: black; color: green
;</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxConexion">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>440</y>
          <width>300</width>
          <height>141</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0)</string>
        </property>
        <property name="title">
         <string>ConexiÃ³n con Hardware</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPuerto">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 0, 0)</string>
           </property>
           <property name="text">
            <string>Puertos</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBoxPuertos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonActualizar">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTip">
              <string>Refrescar puertos</string>
             </property>
             <property name="statusTip">
              <string>Refrescar puertos</string>
             </property>
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelEstado">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Estado</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="editEstado">
           <property name="styleSheet">
            <string notr="true">background color:black;color:green;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="buttonConexion">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Conectar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButtonEnviarConfiguracion">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>220</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 158, 0);</string>
        </property>
        <property name="text">
         <string>Enviar Configuracion</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxConfiguracion">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>211</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 158, 0);</string>
        </property>
        <property name="title">
         <string>Parametros de configuracion</string>
        </property>
        <widget class="QSpinBox" name="spinBoxHumMi">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxHumMa">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="timeEditHoraRie">
         <property name="geometry">
          <rect>
           <x>127</x>
           <y>160</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Humedad MÃ¡nima</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Humedad MÃ¡xima</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tiempo de Riego</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Hora de Riego</string>
         </property>
        </widget>
        <widget class="QTimeEdit" name="timeEditTiempoRiego">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>181</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Estado Actual</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>160</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_Riego">
            <property name="text">
             <string>Riego:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelEstadoActual">
            <property name="text">
             <string>NA</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>160</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Modo:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelModoActual">
            <property name="text">
             <string>NA</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>160</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_LLuvia">
          <item>
           <widget class="QLabel" name="label_Lluvia">
            <property name="text">
             <string>Lluvia:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ELluvia">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>NA</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Graficos">
       <attribute name="title">
        <string>Graficos</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>1201</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
